{"uuid": "cf7bc262-1080-41ff-8cef-48cc78258ca0", "children": ["530978d3-77bc-4d6e-b950-e4e659d6b845"], "befores": [{"name": "create_token", "status": "broken", "statusDetails": {"message": "TypeError: verfiy_http_status_code() got an unexpected keyword argument 'expected_data'\n", "trace": "  File \"/home/jabir/APIAutomation/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/jabir/APIAutomation/.venv/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/jabir/APIAutomation/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/jabir/APIAutomation/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/jabir/APIAutomation/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 913, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/jabir/APIAutomation/tests/test/crud/test_create_full_booking.py\", line 32, in create_token\n    verfiy_http_status_code(response_data=response, expected_data= 200)\n"}, "start": 1712376092810, "stop": 1712376093780}], "start": 1712376092810, "stop": 1712376093837}